_docs/howto/deploy/change-gcp-deployment.md

```diff
@@ -80,7 +91,57 @@ running in, "default", "default".
    property is hardcoded to a specific version. If so, update it to the latest 
    version.
    
-### Run the Deployment Manager update
+
+### Change deployment properties
+
+1. Check `deploy-forseti.yaml.sample` to see if there are any new properties 
+   that you need to copy over to your previously generated 
+   `deploy-forseti-<TIMESTAMP>.yaml`. You can use `git diff` to compare what 
+   changed. For example, to see the diff between the latest (HEAD) and one revision ago:
+
+   ```bash
+   $ git diff origin/master..HEAD~1 -- deploy-forseti.yaml.sample
+   ```
+
+1. Edit `deploy-forseti-<TIMESTAMP>.yaml` and update the values of the new properties.
+
+For example, from v1.1.7 to v1.1.10, the following Compute Engine instance 
+properties have been changed and/or added.
+
+   ```yaml
+   region: $(ref.cloudsql-instance.region)
+
+   network-host-project-id: NETWORK_HOST_PROJECT_ID
+   vpc-name: VPC_NAME
+   subnetwork-name: SUBNETWORK_NAME
+   ```
+
+To upgrade, copy these new properties to your generated 
+deploy-forseti-<TIMESTAMP>.yaml. Then, update the placeholders to the values 
+you want to use. For example, the id of the project that Forseti is 
+running in, "default", "default".
```

Comment:
I think this might be a bit confusing for users since there are 4 variables in the code block.
Can we have something similar to this instead?

region: $(ref.cloudsql-instance.region)

network-host-project-id: __YOUR_PROJECT_ID__
vpc-name: default
subnetwork-name: default

Reply:
done